File: Data/Movie.cs
namespace Trabalho_ISI.Data
{
    public class Movie
    {
        public int Id { get; set; }           // TMDB movie ID
        public string Title { get; set; }
        public string OriginalTitle { get; set; }
        public string Overview { get; set; }
        public string ReleaseDate { get; set; }
        public string PosterPath { get; set; }
        public string BackdropPath { get; set; }
    }
}

File: iOS/TrabalhoISI/TrabalhoISI/Models/Movie.swift
//
//  Movie.swift
//  TrabalhoISI
//
//  Created by Glória Martins on 26/12/2025.
//

import Foundation

struct Movie: Codable, Identifiable {
  var id: Int
  var title: String
  var originalTitle: String?
  var overview: String?
  var releaseDate: String?
  var posterPath: String?
  var backdropPath: String?
}


struct MoviesResponse: Codable {
  let total: Int
  let page: Int
  let pageSize: Int
  let totalPages: Int
  let data: [Movie]
}
